### ISSUE TITLE
molecules can get stuck in the corner

### ISSUE THREAD

**Issue by jbphet** [phetsims/states-of-matter#144]
Sometimes after exploding the container, a molecule will end up stuck in the corner of the container after a while.  The screenshot below shows what this looks like.  In this case, the water molecule first paused at the upper left of the container, then fell down to the lower left and just stayed there, spinning the entire time.

![som-molecule-in-corner](https://cloud.githubusercontent.com/assets/3082831/17908164/941cffa8-693c-11e6-8f4e-d4b3e11e8365.PNG)

**Comment by jbphet** [comment]
This issue was due to some code that was added to try to constrain particles from blowing through the sides of the containers when their velocity was high.  The code ended up just clamping the position, which isn't really what was needed.  I removed the original code and added code to the various Verlet algorithms to detect when the atom or molecule's center of mass would end up outside the container and cause the particle to "bounce".  This required also expanding the container view so that the particles don't appear go beyond the edges.  This has been tested on Chrome and iPad 2 and so far seems to work very well.
